src/Spice86.Core/Emulator/OperatingSystem/DosFileManager.cs

```diff
@@ -776,13 +776,13 @@ public DosFileOperationResult RemoveDirectory(string dosDirectory) {
 
             return DosFileOperationResult.NoValue();
         } catch (IOException e) {
+            triedToDeleteCurrentDir = true;
```

Comment:
Is deleting the currentDir really the only cause for IOExceptions when deleting a directory?